lambdas/build-user-identity/src/test/java/uk/gov/di/ipv/core/builduseridentity/BuildUserIdentityHandlerTest.java

```diff
@@ -248,7 +251,7 @@ void shouldReturnCredentialsWithCiMitVCOnSuccessfulUserInfoRequest() throws Exce
 
     @Test
     void
-            shouldReturnCredentialsWithCiMitVCOnSuccessfulUserInfoRequestAndHasMitigationsFalseAndOnlyNotFoundReturnCodeInCiConfigForAuditEventReturnCodes()
+            shouldReturnCredentialsWithCiMitVCOnSuccessfulUserInfoRequestAndHasMitigationsFalseAndOnlyNotFoundReturnCodeInCiConfigForAuditEventReturnCodesAlongWithTicfVc()
```

Comment:
This test name is getting a bit silly, and is probably indicating that this unit test is covering too many things at the same time...

Can we break this up? Or if you think it should still be one test, then perhaps we should rephrase the name to capture that.

Reply:
done